### PR TITLE
Fix pull reqest events mentioning "Issue" instead of "Pull request"

### LINKED ISSUE #408: Special actions for pull requests often say "Issue" instead of "Pull request"
Examples:

"Issue was closed by ..."
"Issue was referenced by..."

### PR DESCRIPTION
**Pull request by Tunous**
Closes #408.

### CODE REVIEW

File: src/com/gh4a/adapter/IssueEventAdapter.java [hunk 1]
```diff
@@ -192,28 +192,43 @@ protected Intent getIntent() {
         }
     }
 
-    private CharSequence formatEvent(final IssueEvent event, final User user, int typefaceValue) {
+    private CharSequence formatEvent(final IssueEvent event, final User user, int typefaceValue,
+                                     boolean isPullRequestEvent) {
```
Comment (maniac103) [inline src/com/gh4a/adapter/IssueEventAdapter.java]: Nit: indent by 8 spaces, please.

File: src/com/gh4a/loader/IssueCommentListLoader.java [hunk 2]
```diff
@@ -51,13 +51,14 @@ public IssueCommentListLoader(Context context, String repoOwner, String repoName
                 new RepositoryId(mRepoOwner, mRepoName), mIssueNumber);
         List<IssueEvent> events = issueService.getIssueEvents(mRepoOwner, mRepoName, mIssueNumber);
         List<IssueEventHolder> result = new ArrayList<>();
+        boolean isPullRequest = this instanceof PullRequestCommentListLoader;
```
Comment (maniac103) [inline src/com/gh4a/loader/IssueCommentListLoader.java]: I don't like base classes having to know about descendant classes. I think I'd prefer a constructor parameter.

File: src/com/gh4a/loader/IssueCommentListLoader.java [hunk 3]
```diff
@@ -1,25 +1,27 @@
...
 
 public class IssueCommentListLoader extends BaseLoader<List<IssueEventHolder>> {
-    private final String mRepoOwner;
-    private final String mRepoName;
-    private final int mIssueNumber;
+    private final boolean mIsPullRequest;
+
+    protected final String repoOwner;
+    protected final String repoName;
+    protected final int issueNumber;
```
Comment (maniac103) [inline src/com/gh4a/loader/IssueCommentListLoader.java]: All of those should keep the m prefix.

Comment (Tunous) [inline src/com/gh4a/loader/IssueCommentListLoader.java]: Ok, I thought it was just for private fields.

Comment (maniac103) [inline src/com/gh4a/loader/IssueCommentListLoader.java]: No, m is for member variables. I generally follow the AOSP code style: http://source.android.com/source/code-style.html

**Pull request merged**